compiler-plugin/src/main/scala/scalapb/compiler/ProtobufGenerator.scala

```diff
@@ -443,14 +452,32 @@ class ProtobufGenerator(val params: GeneratorParams) extends DescriptorPimps {
         fp.add(s"if (${toBaseType(field)(fieldNameSymbol)} != ${defaultValueForGet(field, true)}) { __size += ${sizeExpressionForSingleField(field, toBaseType(field)(fieldNameSymbol))} }")
       }
     } else if (field.isOptional) {
-      fp.add(s"if ($fieldNameSymbol.isDefined) { __size += ${sizeExpressionForSingleField(field, toBaseType(field)(fieldNameSymbol + ".get"))} }")
+      if (field.customSingleScalaTypeName.isDefined) {
```

Comment:
I might be missing something, but I don't see the difference between the two branches of the if. The first case is assigning `__value` as a temporary variable, the other one inlines the same computation. Why is this necessary?